### PR TITLE
Kerlink firmware and packet forwarder doc update

### PR DESCRIPTION
**Pull request by jpmeijers**
* Update to latest Kerlink firmware
* Switch to Kerlink SPF as packet forwarder
* Clean up documentation

Before this update the docs specified to use the TTN packet forwarder which has not been updated in years. From experience I have seen the stock standard packet forwarder from Kerlink runs much more stable, and it is being kept updated by Kerlink. I have therefore changed the docs to rather use the standard SPF packet forwarder, but with baked in configs for TTN in the dota file.

### CODE REVIEW

**Comment by jpmeijers** [comment]
I just realised I need to make updates to the Kerlink Cellular page too. So one more commit incoming.

**Comment by johanstokking** [comment]
Thanks @jpmeijers! Is this ready to review?

**Comment by jpmeijers** [comment]
@johanstokking it is ready now. My apologies for the notification noise I may have caused you.

File: _content/gateways/kerlink/config.md [hunk 1]
```diff
@@ -9,38 +9,58 @@ zindex: 800
 > 
 > If you have access to the Kerlink wiki it is recommended to follow the firmware upgrade and packet forwarder installation steps from there. This will ensure you are running the latest and most reliable version.
 
-## Configuring packet forwarder
+## Packet forwarders
 
-To configure the gateway, you need to download either [The Things Network's packet forwarder (EU version)](https://github.com/TheThingsNetwork/kerlink-station-firmware/blob/master/dota/dota_thethingsnetwork_v1.3_EU.tar.gz) and [produsb.zip](https://github.com/TheThingsNetwork/kerlink-station-firmware/blob/master/dota/produsb.zip) from our Github.
+Multiple packet forwarders are supported by the Kerlink gateway to forward data to TTN:
+* [Kerlink Semtech Packet Forwarder (SPF)](https://raw.githubusercontent.com/TheThingsNetwork/kerlink-station-firmware/master/dota/dota_spf_3.1.0-klk16_4.1.3-klk8_wirnet_ttn.tar.gz) - uses UDP to forward data to a single LoRaWAN network server
+* [Poly Packet Forwarder](https://github.com/TheThingsNetwork/kerlink-station-firmware/blob/master/dota/dota_thethingsnetwork_v1.3_EU.tar.gz) - can forward to multiple LoRaWAN network servers using UDP
```
Comment (johanstokking) [inline _content/gateways/kerlink/config.md:16]: Shall we stop mentioning this one?

**Review by johanstokking — APPROVED** [review]
cc @mjamescompton @adriansmares https://github.com/TheThingsNetwork/lorawan-stack/issues/1140

**Comment by johanstokking** [comment]
Connecting gateways will be moved to The Things Stack documentation, hence I'm not making comments on small styling/typos as long as it's part of these docs.

If you'd like to contribute to documenting gateways to V3, please report yourself in https://github.com/TheThingsNetwork/lorawan-stack/issues/1140 😉

File: _content/gateways/kerlink/config.md (hunk 1 above)
Comment (jpmeijers) [reply _content/gateways/kerlink/config.md:16]: Seeing as this doc used Poly as the default one until now, maybe better to leave the link here for reference.

**Pull request merged**